test_util/marathon.py

```diff
@@ -290,7 +290,10 @@ def _wait_for_pod_deployment(pod_id):
             r = self.get('v2/pods' + pod_id)
             r.raise_for_status()
             data = r.json()
-            if int(data['scaling']['instances']) != pod_definition['scaling']['instances']:
+            log.debug(data)
```

Comment:
If you want to keep this line, then add some context, e.g. "v2/pods response data: {}".format(data)